example/gardener-local/gardenlet/values-ipv6.yaml

```diff
@@ -0,0 +1,36 @@
+config:
+  gardenClientConnection:
+    bootstrapKubeconfig:
+      kubeconfig: |
+        apiVersion: v1
+        kind: Config
+        current-context: gardenlet-bootstrap
+        clusters:
+        - cluster:
+            insecure-skip-tls-verify: true
+            server: https://kubernetes.default.svc:443 
+          name: default
+        contexts:
+        - context:
+            cluster: default
+            user: gardenlet-bootstrap
+          name: gardenlet-bootstrap
+        users:
+        - name: gardenlet-bootstrap
+          user:
+            token: 07401b.f395accd246ae52d
```

Comment:
Is this needed? I don't see anything IPv6-specific in here?

Reply:
`server: https://kubernetes.default.svc:443` this line changes. Not the internal kubernetes Service gets used for access. cc @einfachnuralex

Reply:
Why was it necessary to change from `https://gardener-local-control-plane:6443` to `https://kubernetes.default.svc:443`?

Reply:
We debugged this issue a little bit more yesterday and it basically boils down to this kind issue i opened https://github.com/kubernetes-sigs/kind/issues/3114 
We don't really know why it does not resolve the kind host when ipv6 is enabled as it works in ipv4 perfectly.

Reply:
Thanks, please add a comment with a link to this issue so that we don't forget about that.